lib/symbol-store.coffee

```diff
@@ -9,34 +9,28 @@ class Symbol
   type: null
 
   constructor: (@text) ->
-    @metadataByPath = {}
+    @metadataByPath = new Map
```

Comment:
Yeah, my understanding is that if they allowed iteration it would expose the internal of the garbage collector in some evil way.